ftdetect/hack.vim

```diff
@@ -11,9 +11,12 @@
 " grant of patent rights can be found in the PATENTS file in the same
 " directory.
 
-au BufRead,BufNewFile *.hhi setl filetype=php
+au BufRead,BufNewFile *.hhi
+  \ setl filetype=hack |
+  \ runtime! syntax/php.vim
```

Comment:
@usox looking at some other issues/PRs here, it seems like you'll know if this is a reasonable way to do this?

Problem is that currently, we overlay a few things on top of the PHP syntax, and we need to keep doing that for now.

Reply:
I really don't like this way of doing this.  It's possible to set what amounts to multiple filetypes by `.`-separating them, so can you just set the filetype to `php.hack` instead of `php` and call it a day?  I don't think trying to strip references to "php" is worth this weird way of handling syntax files.